src/groupchats.c

```diff
@@ -1711,12 +1721,18 @@ static void groupchat_onGroupNickChange(ToxWindow *self, Toxic *toxic, uint32_t
 
     length = MIN(length, TOX_MAX_NAME_LENGTH - 1);
     memcpy(peer->name, new_nick, length);
-    peer->name[length] = 0;
+    peer->name[length] = '\0';
     peer->name_length = length;
 
     line_info_add(self, toxic->c_config, true, peer->prev_name, peer->name, NAME_CHANGE, 0, MAGENTA,
                   " is now known as ");
 
+    ChatContext *ctx = self->chatwin;
+
+    char log_event[TOX_MAX_NAME_LENGTH * 2 + 32];
```

Comment:
No need for `TOX_MAX_NAME_LENGTH * 2` when the previous name is added in write_to_log?

Reply:
True